switchmap/server/db/ingest/ingest.py

```diff
@@ -536,10 +535,10 @@ def insert_macips(items, test=False):
 
     # Insert the values
     if bool(test) is False:
-        _macip.insert_row(rows)
+        _macip.bulk_insert_rows(rows)
```

Comment:
1. Revert.
2. Update `db.add_all` to use the new `bulk_insert_rows` method
3. Apply this to all the` switchmap/server/db/table` files. It's simpler. There v2.0 of sqlalchemy has a `bulk_insert_rows` replacement.